libs/platform/src/lib/dynamic-page/dynamic-page-header/header/dynamic-page-header.component.ts

```diff
@@ -104,7 +104,7 @@ export class DynamicPageHeaderComponent {
 
     /** Collapse/Expand change event raised */
     @Output()
-    collapseChange: EventEmitter<DynamicPageCollapseChangeEvent> = new EventEmitter<DynamicPageCollapseChangeEvent>();
+    collapsedChange: EventEmitter<DynamicPageCollapseChangeEvent> = new EventEmitter<DynamicPageCollapseChangeEvent>();
```

Comment:
I don't think we should really break something
```suggestion
    collapsedChange: EventEmitter<DynamicPageCollapseChangeEvent> = new EventEmitter<DynamicPageCollapseChangeEvent>();
    
    /** @deprecated - use `collapsedChange` instead */
    @Output()
    collapseChange = this.collapsedChange;
```

Reply:
I agree with Dima